lib/ts/recipe/session/recipeImplementation.ts

```diff
@@ -218,7 +222,8 @@ export default function getRecipeInterface(
                 response.session.userId,
                 payload,
                 undefined,
-                response.accessToken !== undefined
+                response.accessToken !== undefined,
+                payload.tId
             );
```

Comment:
In `validateAccessTokenStructure` function in accessToken.ts, we need to check that tId is present if the version is >= 4.

Reply:
Done